metrics/web/test/common/presentation/navigation/metrics_page/metrics_page_factory_test.dart

```diff
@@ -16,153 +18,193 @@ import '../../../../test_utils/route_name_mock.dart';
 
 void main() {
   group("MetricsPageFactory", () {
-    const path = 'path';
+    const unknownRouteName = 'unknown';
+    const pageParameters = DashboardPageParametersModel();
+    const dashboardRouteName = RouteName.dashboard;
+    const projectGroupsRouteName = RouteName.projectGroups;
 
     final metricsPageFactory = MetricsPageFactory();
-    final routeConfiguration = RouteConfigurationMock();
     final routeName = RouteNameMock();
-
-    tearDown(() {
-      reset(routeConfiguration);
-      reset(routeName);
-    });
+    final routeConfiguration = RouteConfigurationMock();
+    final debugMenuPath = RouteConfiguration.debugMenu().path;
+    final dashboardPath = RouteConfiguration.dashboard().path;
 
     test(
-      ".create() returns the dashboard metrics page if the given route configuration is null",
+      ".create() returns the metrics page with the dashboard page child if the given route configuration is null",
       () {
-        final page = metricsPageFactory.create(null);
+        final page = metricsPageFactory.create(null, pageParameters);
 
         expect(page.child, isA<DashboardPage>());
       },
     );
 
     test(
-      ".create() returns the loading metrics page with a name equals to the given route configuration path",
+      ".create() returns the metrics page with the name equals to the route configuration dashboard path if the given route configuration is null",
```

Comment:
🏷️ 
What do you think? To avoid excessive details.
```suggestion
      ".create() returns the metrics page with the name equals to the dashboard path if the given route configuration is null",
```